pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools>=80", "setuptools-scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-cvmfsutils"
```

Comment:
For backwards compatiblity I didn't change it but I'd suggest changing this to:

```suggestion
name = "cvmfsutils"
```

Reply:
Thanks for the contribution, Chris.  Can this change be made without changing the operation of the rpm?

Reply:
I think this metadata only matters for `pip` and I presume we can assume there aren't currently any downstream dependencies, especially given python-cvmfsutils isn't uploaded to PyPI.

Reply:
One thing to consider here would be to name it `cmvfs` instead of `cvmfsutils` to match the name of the import, what do you think?

Reply:
No I think that's too generic of a name.  I wouldn't mind changing the name of the import.  I don't know that anybody is using this library outside of the utilities that are included in it.

Reply:
I think that's a good change.  We'll change the version to 1.0 to indicate an incompatibility.  Maybe it's better as a separate PR from this one though.